src/lib/config/arbitrum/tokenlists.ts

```diff
@@ -2,9 +2,7 @@ import { TokenListURLMap } from '@/types/TokenList';
 
 const tokenlists: TokenListURLMap = {
   Balancer: {
-    Default:
-      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json',
-    Vetted:
+    Allowlisted:
```

Comment:
Since we no longer have more than one, why do we need the nested attribute?

Reply:
I was thinking we'd want the top 100 tokens by volume here. Do we want that list generated on tokenlists update or in the frontend?

Reply:
I don't think we know how we'll fetch that list yet, so maybe don't assume we'll list it here yet? Alternatively, it's fine to leave this like this for now just in case we do need it.